lakey_service/downloader/executors/databricks.py

```diff
@@ -1,13 +1,43 @@
+from databricks_api import DatabricksAPI
+from datetime import datetime
+from django.conf import settings
 
+from .baseexecutor import BaseExecutor
 
-class DataBricksExecutor:
-    # https://docs.databricks.com/api/latest/jobs.html#jobsnotebooktask
 
-    def start_cluster(self):
-        pass
+class DatabricksExecutor(BaseExecutor):
 
-    def start_job(self, spec):
-        pass
+    def __init__(self):
+        self.db = DatabricksAPI(
+            host=settings.DATABRICKS_HOST,
+            token=settings.DATABRICKS_TOKEN
+        )
 
-    def get_job_status(self, job_id):
-        pass
+    def __generate_file_path(self):
```

Comment:
good candidate for the @property.  for example:
```python
@property
def output_file_path(self):
    ...
```

then you don't this below docstring since the name itself is descriptive enough

Reply:
changed, though... isn't it misleading?
It's returning randomized string (see last comment), so it's not 'property' of given executor as such

Reply:
well that's a fair point. Now that you've mentioned it I have my doubts. So how about actually using the method but calling it `get_output_file_path`?

Reply:
:+1: